pypose/lietensor/lietensor.py

```diff
@@ -523,13 +564,15 @@ def identity(cls, *size, **kwargs):
         data = torch.tensor([0., 0., 0., 0., 0., 0., 1., 1.], **kwargs)
         return LieTensor(data.repeat(size+(1,)), ltype=Sim3_type)
 
-    def randn(self, *size, sigma=1, requires_grad=False, **kwargs):
+    def randn(self, *size:_size, sigma:_size_any_t=1.0, requires_grad=False, **kwargs):
         data = sim3_type.Exp(sim3_type.randn(*size, sigma=sigma, **kwargs)).detach()
         return LieTensor(data, ltype=Sim3_type).requires_grad_(requires_grad)
 
     @classmethod
     def add_(cls, input, other):
         return input.copy_(LieTensor(other[..., :7], ltype=sim3_type).Exp() * input)
+    def randn_like(self, *args, sigma=[1.0,1.0,1.0], **kwargs):
```

Comment:
need a space above this line

Reply:
`[1.0, 1.0, 1.0]` instead of `[1.0,1.0,1.0]`

Reply:
done